os/include/debug.h

```diff
@@ -68,14 +68,25 @@
  ****************************************************************************/
 
 #include <tinyara/config.h>
+#ifndef NXFUSE_HOST_BUILD
 #include <tinyara/compiler.h>
 #include <tinyara/logm.h>
+#endif
 
 #include <syslog.h>
 
 /****************************************************************************
  * Pre-processor Definitions
  ****************************************************************************/
+#ifdef NXFUSE_HOST_BUILD
+#define FAR
+#define DEBUGASSERT(x)
+#define ASSERT(x) DEBUGASSERT(x)
+#define OK      0
+#define ERROR   1
```

Comment:
I don't understand why those are necessary. Could you let me know?

Reply:
@sunghan-chang , these are required by linux host machine. If we include types.h of TizenRT, we have to cyclically  include all other dependent header files or make changes in those files. As you see here these variables are defined under NXFUSE_HOST_BUILD and will not disturb target build. We have selectively added this is reduce or avoid unnecessary change to core TizenRT files.